pkg/queue/handler_test.go

```diff
@@ -217,7 +217,7 @@ func TestIgnoreProbe(t *testing.T) {
 	h := ProxyHandler(breaker, stats, false /*tracingEnabled*/, proxy)
 
 	req := httptest.NewRequest(http.MethodPost, "http://prob.in", nil)
-	req.Header.Set(netheader.KubeletProbeKey, "1") // Mark it a probe.
```

Comment:
The queue proxy still handles kubelet probes - so we should leave the test as is

Reply:
Yeah my bad https://github.com/knative/networking/pull/950#issuecomment-2021287510 

I should have woken up before I did my reviews - sorry :/

Reply:
what you had here before is correct